### ISSUE TITLE
Accessibility issue: Frames must have an accessible name on /speaking.html

### ISSUE THREAD

**Issue by willingc** [willingc/willing-consulting-2022#17]
## What
An accessibility scan flagged the element `<iframe data-external="1" src="https://www.youtube.com/embed/1ejP3wjIAeE" title="" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>` on https://willingconsulting.com/speaking.html because frames must have an accessible name. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.10/frame-title?application=playwright.

To fix this, ensure <iframe> and <frame> elements have an accessible name.

Specifically:

Fix any of the following:
-   Element has an empty title attribute
-   aria-label attribute does not exist or is empty
-   aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-   Element&apos;s default semantics were not overridden with role="none" or role="presentation"

## Acceptance Criteria
- [ ] The specific axe violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
